libs/nspr/patches/009-rm_srcdir_from_configure_in.patch

```diff
@@ -0,0 +1,19 @@
+the $srcdir is not defined at the time of gnu-configurize.
+
+Upstream-Status: Inappropriate [OE-Core specific]
```

Comment:
is this one needed? is not already the sort of thing that PKG_FIXUP handles?

Reply:
yes, I think some of them can be dropped, I'll test compile on more targets